sql_tasks/README.md

```diff
@@ -0,0 +1,225 @@
+## Create tables
+
+#### `Product`
+
+| Field | Type |
+| ----- | ---- |
+| sku | str |
+| status_id | int |
+| name | str |
+| taxable | bool |
+| price | int |
+| created_at | datetime |
+
+#### `ProductStatus`
+
+| Field | Type |
+| ----- | ---- |
+| name | str |
+
+```sql
+CREATE TABLE productstatus (
+	id serial primary key,
+	name varchar(10)		
+);
+
+CREATE TABLE product (
+	id serial primary key,
+	sku varchar(10),		
+	status_id integer references productstatus(id),
+	name varchar(10),		
+	tax BOOLEAN,			
+	price integer,			
+	created_at timestamp	
+);
+```
+
+## Queries
+
+#### 1. Добавить запись в таблицу `Product`
+
+```sql
+INSERT INTO product(sku, status_id, name, tax, price, created_at) 
+VALUES (10, 1, 'fish', TRUE, 5, '2019-06-26 13:15:20');
+```
+
+#### 2. Добавить запись в таблицу `ProductStatus`
+
+```sql
+INSERT INTO productstatus(name) VALUES ('confirmed');
+```
+
+#### 3. Добавить 4 записи в таблицу `ProductStatus`
+
+```sql
+INSERT INTO productstatus(name) VALUES 
+    ('confirmed'), 
+    ('placed'), 
+    ('delivered'), 
+    ('paid');
+```
+
+#### 4. Добавить 500 записей в таблицу `Product` (название, текст не важен)
+
+```sql
+DO $$
+declare id integer :=2;
+BEGIN
+	WHILE id > 0 AND id < 500 LOOP	
+		INSERT INTO product(sku, status_id, name, tax, price, created_at)  
+		values (10, 3, 'Dima', True, 10, current_timestamp);
+		id = id + 1;
+	END LOOP;
+END $$;
+```
+
+#### 5. Обновить название одной из записей в таблице `ProductStatus`
+
+```sql
+UPDATE productstatus
+SET name = 'available'
+WHERE
+	name = 'paid';
+```
+
+#### 6. Обновить значение `taxable` в первых 10 записях таблицы `Product`
+
+```sql
+update product
```

Comment:
How about this trick?
Postgres docs recommend to use array instead of IN and subquery. This should work much faster
UPDATE product
SET tax = true
WHERE id = any (array(SELECT id FROM product ORDER BY id LIMIT 10));